lib/data-types.d.ts

```diff
@@ -396,105 +512,128 @@ export class VIRTUAL<T> extends ABSTRACT {
  *
  * @property ENUM
  */
-export class ENUM<T extends string> extends ABSTRACT {
-  constructor(...values: string[]);
-  constructor(options: ENUM_Options<T>);
-  values: T[];
-  options: ENUM_Options<T>;
+export const ENUM: EnumDataTypeStatic;
+
+export interface EnumDataTypeStatic extends AbstractDataTypeStatic {
```

Comment:
the reason I had `T` before is to support string literal types. i.e. 
```ts
type SomeEnum = 'a' | 'b' | 'c';
Sequelize.ENUM<SomeEnum>('a', 'b', 'c', 'd'); // would throw an error because of 'd'
```